app/assets/stylesheets/components/_forms.scss

```diff
@@ -1,7 +1,5 @@
 input {
-  border-style: solid;
-  border-width: 1px;
-  border-color: rgb(227, 227, 227);
+  border: 1px solid rgb(227, 227, 227);
```

Comment:
Properties should be ordered background-color, border, border-radius, box-shadow, font-size, height, line-height, outline, padding